### DISCUSSION TITLE
Lesson 1: is ETH PoW or PoS ?

### DISCUSSION THREAD

**Discussion by devmousa** [Cyfrin/foundry-full-course-cu#493] [Q&A] [answered]
**Hello everyone and I hope you are all doing well**

I'm learning web3 from Mr. Patrick Collins course
when I reached high-level blockchain fundamentals, Mr. Patrick wrote that ETH is PoW since _15 SEP 2022_

but after a little bit time, he wrote that ETH is PoS

I'm really confused 😅
which one is the right ?
PoW or PoS ? (AFAIK it's PoS since ETH2.0 was released) 

and thank you all in advance ❤️

**Comment by Niferu** [comment, marked as answer]
Hello @devmousa 

Ethereum officially switched to a Proof of Stake (PoS) consensus mechanism in 2022 as a more secure and energy-efficient way to validate transactions and add new blocks to the blockchain. ETH is PoS now, but previously it was PoW.

**Reply by devmousa** [reply]
Thank you so much @Neftyr
